Library/PasswordValidation.swift

```diff
@@ -0,0 +1,25 @@
+import Foundation
+
+public typealias Password = String
+
+public func passwordsMatch(_ pwds: (first: Password, second: Password)) -> Bool {
```

Comment:
Couple convenience logic around matching passwords (reused within `CreatePasswordVM` & `ChangePasswordVM`.

Reply:
Wonder if we can simply do a `String` comparison...then the call site would look as simple as `let validationMatch = combinedPasswords.map(=)`